src/ert/shared/models/base_run_model.py

```diff
@@ -530,3 +531,41 @@ def check_if_runpath_exists(self) -> bool:
                 if Path(run_path).exists():
                     return True
         return False
+
+    def validate(self) -> None:
+        if self._simulation_arguments is None:
+            return
+        errors = []
+
+        current_case = self._simulation_arguments.get("current_case", None)
+        target_case = self._simulation_arguments.get("target_case", None)
+
+        if current_case is not None and current_case in self._ert.storage_manager:
+            if (
+                len(self._ert.storage_manager[current_case].getStateMap())
+                < self._ert._ensemble_size
+            ):
+                errors.append(
+                    f"ERROR: Existing case {current_case} was created with ensemble "
+                    f"size smaller than specified in the ert configuration file ("
+                    f"{len(self._ert.storage_manager[current_case].getStateMap())} "
+                    f" < {self._ert._ensemble_size})"
+                )
+        if target_case is not None:
+            if target_case == current_case:
+                errors.append(
+                    f"ERROR: Target case and current case can not have the same name. "
+                    f"They were both: {current_case}"
+                )
+
+            if "%d" in target_case:
+                num_iterations = self._simulation_arguments["num_iterations"]
+                for i in range(num_iterations):
+                    if target_case % i in self._ert.storage_manager:
+                        errors.append(f"ERROR: Target case {target_case % i} exists")
+
+            elif target_case in self._ert.storage_manager:
+                errors.append(f"ERROR: Target case {target_case} exists")
```

Comment:
Suggest removing `ERROR` and making it the name of the window instead, also adding a `:` after: `Target case: ` would make it easier to tell what the case name is (espesially as `default` is the default)